vllm/_custom_ops.py

```diff
@@ -13,16 +12,8 @@
 
 logger = init_logger(__name__)
 
-if not current_platform.is_tpu() and not current_platform.is_xpu():
-    try:
-        import vllm._C
-    except ImportError as e:
-        logger.warning("Failed to import from vllm._C with %r", e)
-
-supports_moe_ops = False
-with contextlib.suppress(ImportError):
-    import vllm._moe_C  # noqa: F401
-    supports_moe_ops = True
+current_platform.import_core_kernels()
+supports_moe_ops = current_platform.try_import_moe_kernels()
```

Comment:
this attribute is only used here:

https://github.com/vllm-project/vllm/blob/17edd8a807019c8d1e58634aecb1de7984e8d467/vllm/_custom_ops.py#L1539

I wonder if we can remove `supports_moe_ops` and just do:

```python
if hasattr(torch.ops, "_moe_C") and hasattr(torch.ops._moe_C, "marlin_gemm_moe"):
```

then having a simple `import_kernels` interface for the platform class would sounds better.

cc @NickLucche

Reply:
@ProExpertProg do you know why we don't use `if hasattr(torch.ops, "_moe_C") and hasattr(torch.ops._moe_C, "marlin_gemm_moe"):` in the first place? or maybe @tlrmchlsmth @bnellnm have better ideas.

Reply:
Yeah ++ that seems simpler